python/py-pyqt5/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup               python 1.0
 
 name                    py-pyqt5
```

Comment:
to further simplify the portfile, please add ```python.rootname     PyQt5``` under the ```name``` field (you can then remove a few more lines below)